server/src/main/java/org/opensearch/index/shard/RemoteStoreRefreshListener.java

```diff
@@ -60,7 +61,7 @@
  */
 public final class RemoteStoreRefreshListener implements ReferenceManager.RefreshListener {
 
-    private static final Logger logger = LogManager.getLogger(RemoteStoreRefreshListener.class);
+    private final Logger logger;
```

Comment:
if there are too many shards on a node, can this cause any degradation on any resource?

Reply:
its hard to say with full confidence as there would more logger objects now. But IMO this should not cause any issues as its just a simple logger object.